### DISCUSSION TITLE
Installation without Composer

### DISCUSSION THREAD

**Discussion by xguntis** [dvdoug/BoxPacker#233] [General]
After downloading the code from Github, what should I do to install the library into my code?

**Comment by dvdoug** [comment]
Hi @xguntis 

I don't know how to say this without appearing a little rude so apologies for that, but if you don't understand how PHP autoloading works then perhaps you really should be using Composer. Is there a reason you don't want to?

Otherwise https://www.php-fig.org/psr/psr-4/examples/ may assist
